src/ontology/mondo-edit.obo

```diff
@@ -440940,6 +440940,23 @@ synonym: "VLS" EXACT ABBREVIATION [PMID:32923201]
 is_a: MONDO:0005275 {source="PMID:32923201"} ! lung disease
 created_by: http://orcid.org/0000-0002-5460-8025
 
+[Term]
+id: MONDO:0600030
+name: B-cell acute lymphoblastic leukemia with t(1;19)(q23;p13.3); E2A-PBX1 (TCF3-PBX1)
+def: "A B-cell acute leukemia characterized by the presence of lymphoblasts that carry a translocation between the E2A gene on chromosome 19 and the PBX1 gene on chromosome 1. It occurs in children and less often in adults." [NCIT:C80347]
+synonym: "B acute lymphoblastic leukemia with t(1;19)(q23;p13.3); E2A-PBX1 (TCF3-PBX1)" EXACT [] {source="NCIT:C80347"}
```

Comment:
synonyms should have database cross references, not source. These all need to be fixed